### DISCUSSION TITLE
Vidsrc.to popup ad blocking

### DISCUSSION THREAD

**Discussion by TheFilthyFrog** [AdguardTeam/AdGuardHome#6869] [Q&A]
when i first started using adguard to block popups on vidsrc.to is was working awesome, however over the last month or so it seems there has been more and more popups showing again.  the original user filters i found on the internet related to vidsrc.to were:

vidsrc.to ##body > div:nth-of-type(2)

is their any other custom filters people are using for vidsrc.to?
Thanks

**Comment by ghost** [comment]
Hi!

You can report website ads using the [AdGuard reporting tool](https://adguard.com/kb/guides/report-website/).
